apps/project/models.py

```diff
@@ -13,6 +14,8 @@
 from category_editor.models import CategoryEditor
 from project.permissions import PROJECT_PERMISSIONS, PROJECT_PERMISSION_MODEL_MAP
 from organization.models import Organization
+from entry.models import Entry
+from lead.models import Lead
```

Comment:
Let's import this inside with NOTE: to avoid circular import. And undo changes in other files.